spec/rails_helper.rb

```diff
@@ -69,4 +69,7 @@
   # Allow request specs to use have_* matchers, like so:
   # expect( response.body ).to have_title 'My Page Title'
   config.include Capybara::RSpecMatchers, type: :request
+
+  # Load my app's helpers
+  config.include FeaturesHelper, type: :request
```

Comment:
Try to avoid including app/ helpers in specs, feels risky, it might lead to some unexpected method overrides. Perhaps this can be removed if i18n variable interpolation is used?

To test a helper, this can be done in `spec/helpers/`, in a dedicated spec.